### DISCUSSION TITLE
国庆更新

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#360] [通知]
## 🚀 功能

- 添加播放速率选择 @Richasy (#358)
- 添加动态的本机分享 @Richasy (#353)
- 支持互动视频 @Richasy (#350)
- 添加侧键返回/支持系统返回按键 @Richasy (#344)
- 增加下载接口选择 @Richasy (#341)
- 支持显示联合创作 @Richasy (#335)
- 添加字幕 @Richasy (#327)

## 🐛 Bug 修复

- 修复连载索引显示错误的问题 @Richasy (#356)
- 修复点击关注直播间弹出层未关闭的问题 @Richasy (#354)
- 修复专栏阅读滚动问题 @Richasy (#351)
- 修复全屏/全窗口切换时的问题 @Richasy (#342)
- 观看自己的视频时不显示关注按钮 @Richasy (#332)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2110.1.0'>国庆更新</a>.</em>

**Comment by whpu2020** [comment]
Add-AppxPackage : 部署失败，原因是 HRESULT: 0x80070005, 拒绝访问。

错误 0x80070005: 从位置 App_2.2109.2801.0_x64.msixbundle 中打开程序包失败。

注意: 有关其他信息，请在事件日志中查找 [ActivityId] da6f23ca-bab9-0002-6f4b-72dab9bad701，或使用命令行 Get-AppPackageLo
g -ActivityID da6f23ca-bab9-0002-6f4b-72dab9bad701

所在位置 D:\Bili.Uwp_2.2109.2801.0_x64\App_2.2109.2801.0_Test\Add-AppDevPackage.ps1:487 字符: 13
+             Add-AppxPackage -Path $DeveloperPackagePath.FullName -Dep ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (D:\Bili.Uwp_2.2..._x64.msixbundle:String) [Add-AppxPackage]，PSSecurityEx
    ception
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

错误: 未能安装应用程序。
